hw2/05-form.html

```diff
@@ -7,13 +7,103 @@
     <title>05 - Form</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="05-form.js"></script>
   </head>
-  <body>
+  <body class="d-flex justify-content-center bg-dark">
     <main>
-      <form>
-        <h1>Exercise 05 - Form</h1>
+      <form
+        class="bg-light border border-light rounded w-75 mx-auto p-5 m-5"
+        id="form"
+      >
+        <h1 class="text-center">Exercise 05 - Form</h1>
+        <div class="form-group row p-1 mt-4">
+          <label class="row p-1 fw-bold" for="fullname">Full Name: </label>
+          <input
+            type="text"
+            name="fullname"
+            id="fullname"
+            class="row p-1 form-control"
+          />
+        </div>
+        <div class="form-group row p-1">
+          <label class="row p-1 fw-bold" for="email">Email: </label>
+          <input
+            type="email"
+            name="Email"
+            id="email"
+            class="row p-1 form-control"
+          />
+        </div>
+        <div class="form-group row p-1">
+          <label class="row p-1 fw-bold" for="registration"
+            >Registration Status:
+          </label>
+          <select
+            class="form-select row p-1"
+            id="registration"
+            name="registration"
+          >
+            <option selected>Choose...</option>
+            <option value="1">registered</option>
```

Comment:
In the console, these values are displayed as numbers which aren't too descriptive if we sent this info to someone without context of the title in the dropdown.

Reply:
good catch! fixed.